docker-compose.yml

```diff
@@ -8,14 +8,13 @@ services:
       BROKER_URL: amqp://guest:guest@rabbitmq:5672
       MONGO_URL: mongodb://mongo:27017
     working_dir: /app/coinstacks/${COINSTACK}/api
-    command: dockerize -wait tcp://mongo:27017 -timeout 5m -wait-retry-interval 10s sh -c "yarn nodemon"
+    command: yarn nodemon
```

Comment:
not waiting for mongo anymore?

Reply:
Removed the register/unregister endpoints at this time since they aren't being used. Will most likely be adding back in some way come websocket subscription

Reply:
still needed for block height though yeah?

Reply:
In the ingester, this is just the dockerize wait for the API service